src/strands/models/gemini.py

```diff
@@ -40,10 +64,16 @@ class GeminiConfig(TypedDict, total=False):
             params: Additional model parameters (e.g., temperature).
                 For a complete list of supported parameters, see
                 https://ai.google.dev/api/generate-content#generationconfig.
+            gemini_tools: Gemini-specific tools that are not FunctionDeclarations
+                (e.g., GoogleSearch, CodeExecution, ComputerUse, UrlContext, FileSearch).
+                Use the standard tools interface for function calling tools.
+                For a complete list of supported tools, see
+                https://ai.google.dev/api/caching#Tool
         """
 
         model_id: Required[str]
         params: dict[str, Any]
+        gemini_tools: list[genai.types.Tool]
```

Comment:
@pgrayy thoughts on naming here? Would we want to be consistent among providers - maybe something like built_in_tools?

Reply:
I think we should punt on this for now.  If we eventually unify on `built_in_tools` we might want a standardized type to be used among providers whereas as-is this unblocks built-in tools.

We can always add getters as needed